<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="TItuloLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Perpetua Titling MT</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kinova Gen 3 Lite</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>242</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="ConectarLO">
     <item row="0" column="0">
      <widget class="QPushButton" name="ConectarPB">
       <property name="text">
        <string>Conectar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="MarkConeccion">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Conecci√≥n exitosa</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>368</width>
      <height>287</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="GananciasLO">
     <item row="4" column="3">
      <widget class="QDoubleSpinBox" name="KD4">
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="KI1">
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QDoubleSpinBox" name="KI6">
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="KP3">
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="KP6">
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QDoubleSpinBox" name="KI4">
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="Actuador2Label">
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>         2:        </string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="Actuador6Label">
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>         6:        </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="KP2">
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QDoubleSpinBox" name="KD2">
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="KI2">
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="KP5">
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Actuador1Label">
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>         1:        </string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QDoubleSpinBox" name="KD3">
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QDoubleSpinBox" name="KI3">
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QDoubleSpinBox" name="KD5">
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QDoubleSpinBox" name="KD6">
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="KILabel">
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>     KI</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDoubleSpinBox" name="KD1">
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="KP4">
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="KPLabel">
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>     KP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="KP1">
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="KDLabel">
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>     KD</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="Actuador4Label">
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>         4:        </string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QDoubleSpinBox" name="KI5">
       <property name="maximum">
        <double>100.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="ActuadorLabel">
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>  Actuador</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="Actuador3Label">
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>         3:        </string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="Actuador5Label">
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>         5:        </string>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>Actuador6Label</zorder>
    <zorder>KILabel</zorder>
    <zorder>KPLabel</zorder>
    <zorder>KD2</zorder>
    <zorder>KDLabel</zorder>
    <zorder>Actuador1Label</zorder>
    <zorder>KP3</zorder>
    <zorder>KD5</zorder>
    <zorder>KP6</zorder>
    <zorder>Actuador2Label</zorder>
    <zorder>Actuador5Label</zorder>
    <zorder>Actuador3Label</zorder>
    <zorder>KP5</zorder>
    <zorder>KI5</zorder>
    <zorder>KI3</zorder>
    <zorder>ActuadorLabel</zorder>
    <zorder>Actuador4Label</zorder>
    <zorder>KD1</zorder>
    <zorder>KI2</zorder>
    <zorder>KI1</zorder>
    <zorder>KD4</zorder>
    <zorder>KI6</zorder>
    <zorder>KI4</zorder>
    <zorder>KD3</zorder>
    <zorder>KD6</zorder>
    <zorder>KP1</zorder>
    <zorder>KP2</zorder>
    <zorder>KP4</zorder>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>120</y>
      <width>191</width>
      <height>90</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="EjecutarLO">
     <item row="0" column="0">
      <layout class="QGridLayout" name="IniciarLO">
       <item row="0" column="0">
        <widget class="QLabel" name="EjecutarLabel">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Ejecutar</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="IniciarPB">
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <layout class="QGridLayout" name="DetenerLO">
       <item row="0" column="0">
        <widget class="QLabel" name="DetenerLabel">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Detener</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="DetenerPB">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>204</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="TiempoLO">
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="CambiarTiempoPB">
       <property name="text">
        <string>Cambiar</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="SegundosLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Segundos</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSpinBox" name="TiempoS"/>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="TALabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tiempo de Accion</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>PD + CancG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>P'D' + CancG</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>S</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>202</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="ControladoresLO">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QRadioButton" name="PIDSel">
         <property name="text">
          <string>PID Lineal</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="PDCancGSel">
         <property name="text">
          <string>PD+ CancG</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="PDComGSel">
         <property name="text">
          <string>PD+CompG</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QRadioButton" name="PdCompGSel">
         <property name="text">
          <string>P'D'+Comp</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QRadioButton" name="PdCancGSel">
         <property name="text">
          <string>P'D'+Canc</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="PDSel">
         <property name="text">
          <string>PD Lineal</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="1" column="0">
        <widget class="QRadioButton" name="PIDSel_2">
         <property name="text">
          <string>PID Lineal</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="PDCancGSel_2">
         <property name="text">
          <string>PD+ CancG</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QRadioButton" name="PDComGSel_2">
         <property name="text">
          <string>PD+CompG</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QRadioButton" name="PdCompGSel_2">
         <property name="text">
          <string>P'D'+Comp</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QRadioButton" name="PdCancGSel_2">
         <property name="text">
          <string>P'D'+Canc</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="PDSel_2">
         <property name="text">
          <string>PD Lineal</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>230</y>
      <width>258</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="ErroresLO">
     <item row="0" column="0">
      <widget class="QLabel" name="ErroresLabel">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Errores</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTextBrowser" name="errorTextBrowser"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>914</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuREADME">
    <property name="title">
     <string>README</string>
    </property>
   </widget>
   <addaction name="menuREADME"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>ConectarPB</tabstop>
  <tabstop>PDSel</tabstop>
  <tabstop>PIDSel</tabstop>
  <tabstop>PDCancGSel</tabstop>
  <tabstop>PDComGSel</tabstop>
  <tabstop>PdCancGSel</tabstop>
  <tabstop>PdCompGSel</tabstop>
  <tabstop>PDSel_2</tabstop>
  <tabstop>PIDSel_2</tabstop>
  <tabstop>PDCancGSel_2</tabstop>
  <tabstop>PDComGSel_2</tabstop>
  <tabstop>PdCancGSel_2</tabstop>
  <tabstop>PdCompGSel_2</tabstop>
  <tabstop>KP1</tabstop>
  <tabstop>KP2</tabstop>
  <tabstop>KP3</tabstop>
  <tabstop>KP4</tabstop>
  <tabstop>KP5</tabstop>
  <tabstop>KP6</tabstop>
  <tabstop>KI1</tabstop>
  <tabstop>KI2</tabstop>
  <tabstop>KI3</tabstop>
  <tabstop>KI4</tabstop>
  <tabstop>KI5</tabstop>
  <tabstop>KI6</tabstop>
  <tabstop>KD1</tabstop>
  <tabstop>KD2</tabstop>
  <tabstop>KD3</tabstop>
  <tabstop>KD4</tabstop>
  <tabstop>KD5</tabstop>
  <tabstop>KD6</tabstop>
  <tabstop>TiempoS</tabstop>
  <tabstop>CambiarTiempoPB</tabstop>
  <tabstop>IniciarPB</tabstop>
  <tabstop>DetenerPB</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>MarkConeccion</tabstop>
  <tabstop>errorTextBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
